content/news/devblog-20200629.md

```diff
@@ -0,0 +1,64 @@
+
+OpenRA's game engine has been developed from scratch over many years using different programming languages and principles to the original C&Cs. This means that we cannot *directly* copy code from the source release, but instead use it as a reference source to learn what made the original games tick and to translate concepts and implementation details across where appropriate to improve OpenRA.
+
+The next OpenRA release will contain some initial small fixes based on what we have already learned. One example of this is the original palette remap tables that the original Red Alert used to create the different civilian infantry types that were previously missing from OpenRA's version. There is a lot more to learn, and we expect that this knowlege will influence and inspire many details in the evolution and growth of the OpenRA games over the next few years.
+
+A popular question has been whether OpenRA will add support the remastered game assets, and I can now confirm (see [this GitHub comment](https://github.com/OpenRA/OpenRA/issues/18239#issuecomment-646380281) for details) that we will work towards this goal. This will take time, as there are a number of technical challenges that must first be solved. We will provide more information on these plans in future posts, or you can follow progress as we proceed on GitHub.
+
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20200629-selection.png" width="600" alt="Tiberian Sun selection boxes">
+
+Structures in Tiberian Sun now use the correct isometric-style selection boxes.
+
+</div>
+
+In the meantime, work has continued on OpenRA's version of Tiberian Sun. Visible progress has been slow over the last couple of years while we focused on rebuilding parts of the low-level game engine to add the flexibility required to support the "gen2" games (Tiberian Sun, Red Alert 2, and community mods based on them), without breaking support for "gen1" games (Tiberian Dawn, Red Alert, Dune 2000 and existing community mods). This work is now starting to pay off, as we are able to leverage the added flexibilty to enable some long-awaited features.
+
+The first of these are the isometric selection boxes used by structures. This may sound like a simple cosmetic change, but behind the scenes many changes were required to replace code designed for simple rectangular regions to work efficiently with more complicated shapes (such as the hexagons used by isometric buildings). The engine-level changes were actually added back in [release-20180218](/news/release-20180218/), but we have only recently had time to adapt the game code that defines and draws the selection boxes to make use of it. OpenRA's isometric selection boxes have some visual tweaks compared to the original game to better integrate with OpenRA's dynamic zooming and dynamic progress bars.
```

Comment:
I'm not too happy with the wording of this paragraph, but don't yet have a specific idea on how to fix it.